l10n_it_vat_statement_split_payment/models/statement.py

```diff
@@ -71,6 +71,16 @@ def compute_amounts(self):
                         }
                     )
 
+                    # Interests on total_vat_statement without split lines
```

Comment:
Questa modifica cambia il comportamento del modulo quindi non è un refactoring, puoi spostarlo in un commit dedicato? Sarebbe bene anche includere un test.

Inoltre questo codice è in buona parte già presente in https://github.com/OCA/l10n-italy/blob/4c19d383ff3fca307a69a67515e864827a84cb5e/account_vat_period_end_statement/models/account.py#L700-L709, è possibile non duplicarlo?
A proposito: questo valore non dovrebbe essere già in `interests_debit_vat_amount`? Come mai serve rifare lo stesso calcolo?

Perdipiù è una modifica a un altro modulo, potresti metterla in un commit dedicato come indicato nelle linee guida https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#71commit-message?
> Try to split into different commits where impacted modules are different

Reply:
spostato in altro commit